src/hb-ot-shape.cc

```diff
@@ -116,7 +116,8 @@ hb_ot_shape_planner_t::compile (hb_ot_shape_plan_t &plan,
     plan.fallback_mark_positioning = true;
 
   /* Currently we always apply trak. */
-  plan.apply_trak = hb_aat_layout_has_tracking (face);
+  plan.apply_trak = hb_aat_layout_has_tracking (face) &&
+		    plan.map.needs_fallback (HB_TAG ('t','r','a','k'));
```

Comment:
This change is not good.  It will disable trak table if GSUB or GPOS has trak.  Not sure which one.